packages/cli/src/ui/components/__snapshots__/InputPrompt.test.tsx.snap

```diff
@@ -32,6 +32,30 @@ exports[`InputPrompt > command search (Ctrl+R when not in shell) > renders match
  git commit -m "feat: add search" in src/app"
 `;
 
+exports[`InputPrompt > queued message editing > should navigate input history on fresh start when no queued messages exist 1`] = `
```

Comment:
these snapshots don't show anything. they all are just the initial UI. is that intentional?